themes/default/content/product/_index.md

```diff
@@ -12,80 +12,115 @@ subheading: |
+                    Use your favorite IDEs with autocomplete, in-line documentation, and a natural development experience.
 
-              - title: Create reusable infrastructure
+              - title:
                 icon: puzzle
                 icon_color: yellow
                 description:
-                    Build and share cloud infrastructure with reusable [Pulumi Packages](/product/packages/), which
-                    encapsulate your cloud architectures and best practices.
+                    Run rapid unit tests with any supported framework to validate your infrastructure code before deployment.
 
-        - description: Deploy cloud infrastructure and applications together
+        - description: Consume infrastructure as code in any language
           features:
-              - title: Multiple deployment options
+              - title:
                 icon: rocketship
                 icon_color: salmon
                 description: |
-                    Deploy infrastructure interactively with a CLI, programmatically with Pulumi’s
-                    [Automation API](/automation/), or through your [CI/CD process](/docs/guides/continuous-delivery/).
+                    Build faster by reusing Pulumi libraries and CDK constructs, and create your own libraries as multi-language Pulumi Packages.
 
-              - title: Deployments as Code
+              - title:
                 icon: gear
                 icon_color: salmon
                 description: |
-                    Run deployments from your application code at runtime with [Automation API](/automation/). Create infrastructure APIs, custom platforms, and CLIs.
+                    Develop modern cloud applications faster while using well-architected best practices with the Crosswalk for AWS library.
 
-              - title: Preview and test changes
+              - title:
                 icon: eye
                 icon_color: salmon
                 description: |
-                    Test and validate infrastructure with standard [unit test frameworks](/docs/guides/testing/#unit-testing) and
-                    [integration tests](/docs/guides/testing/integration/). Preview changes before deploying.
+                    Convert your existing infrastructure code to Pulumi (TF, ARM, CFN, K8s, CRD, Helm) and import existing resources into Pulumi code.
+     
+        - description: Deploy to over 100 cloud and SaaS providers
+          features:
+              - title: 
+                icon: shield
+                icon_color: purple
+                description: |
+                    Native providers for AWS, Azure, Google Cloud, and Kubernetes have 100% API coverage and same-day updates for new features.
+
+              - title: 
+                icon: security
+                icon_color: purple
+                description: |
+                    Use a single workflow to manage cloud native infrastructure like Kubernetes, Helm, CRDs, GitOps, and more.
```

Comment:
link kubernetes to https://www.pulumi.com/registry/packages/kubernetes/
link helm to https://www.pulumi.com/blog/helm-release-resource-for-kubernetes-generally-available/
link GitOps to https://www.pulumi.com/blog/pulumi-kubernetes-new-2022/
link CRDs to https://www.pulumi.com/blog/new-kubernetes-superpowers/#strongly-typed-kubernetes-crds